CodeMaid/UI/Dialogs/Options/Reorganizing/ReorganizingGeneralDataTemplate.xaml

```diff
@@ -15,6 +15,18 @@
                         <RadioButton Content="type then access" IsChecked="{Binding PrimaryOrderByAccessLevel, Converter={x:Static cnv:BooleanInverseConverter.Default}}" />
                         <RadioButton Content="access then type" IsChecked="{Binding PrimaryOrderByAccessLevel}" />
                     </StackPanel>
+                    <StackPanel
```

Comment:
More nitpicks:

* Can we follow the formatting conventions of the rest of the file (e.g. attributes on same line)
* Can we use the same verbiage as other options.. (e.g. "Access levels should be ordered by" with the default option being first "public to private" and the alternate option being second "private to public")